travis.yml

```diff
@@ -0,0 +1,26 @@
+language: node_js
+node_js:
+  - "8.9.4"
+
+branches:
+  only:
+    - master
+
+before_script:
+  - npm install -g @angular/cli
+
+script:
+  - ng build --prod --bh
```

Comment:
this needs a value

Reply:
not if I want to change from "/" to "" I just tried it It's working